packages/wagmi/src/utils/helpers.ts

```diff
@@ -1,46 +1,15 @@
-import type { WindowProvider } from '@wagmi/core'
-import type { CaipNetwork, CaipNetworkId, Tokens } from '@web3modal/scaffold'
-import type { Web3ModalClientOptions } from '../client.js'
-import { NAMESPACE } from './constants.js'
-import { NetworkImageIds } from './presets.js'
+import type { CaipNetwork } from '@web3modal/scaffold'
+import type { Chain } from '@wagmi/core'
+import { ConstantsUtil, PresetsUtil } from '@web3modal/utils'
 
-export function getCaipDefaultChain(chain?: Web3ModalClientOptions['defaultChain']) {
+export function getCaipDefaultChain(chain?: Chain) {
   if (!chain) {
     return undefined
   }
 
   return {
-    id: `${NAMESPACE}:${chain.id}`,
+    id: `${ConstantsUtil.NAMESPACE}:${chain.id}`,
     name: chain.name,
-    imageId: NetworkImageIds[chain.id]
+    imageId: PresetsUtil.NetworkImageIds[chain.id]
```

Comment:
Presets should point to a more concrete data set now, i.e. `EIP155NetworkImageIds` etc.., try to get into the mindset where you are working on solana implementation in the future and will be defining new presets / constants etc in utils package.